entity-services/src/test/resources/model-units/instance-converter.xqy

```diff
@@ -235,37 +274,8 @@ declare function et-required:instance-to-envelope(
     $entity-instance as map:map
 ) as document-node()
 {
-    et-required:instance-to-xml-envelope($entity-instance)
+    et-required:instance-to-envelope($entity-instance, "xml")
```

Comment:
@grechaw I think it will be good to add a comment somewhere here to note that this function is being kept for backwards compatibility